src/views/Loading/Loading.jsx

```diff
@@ -1,44 +1,24 @@
 import React from 'react'
-import { connect } from 'react-redux'
-import { withRouter } from 'react-router'
-import { i18n } from '../../config'
-import { getApplicationState } from '../../actions/ApplicationActions'
-import { Spinner, SpinnerAction } from '../../components/Form'
-import { timeout } from '../../components/Form/Location/Location'
+import i18n from 'util/i18n'
+import { Spinner, SpinnerAction } from 'components/Form'
+import { timeout } from 'components/Form/Location/Location'
 
 class Loading extends React.Component {
   constructor(props) {
     super(props)
 
     this.state = {
-      spinner: this.props.spinner,
-      spinnerState: SpinnerAction.ACTION_SPIN
-    }
-
-    // Animations
-    this.animationDelay = null
-    this.cancelAnimations = this.cancelAnimations.bind(this)
-    this.animateCloseTimeout = this.animateCloseTimeout.bind(this)
-  }
-
-  componentWillMount() {
-    if (!this.props.authenticated) {
-      this.props.history.push('/login')
+      spinner: false,
+      spinnerAction: SpinnerAction.ACTION_SPIN,
     }
   }
 
   componentDidMount() {
-    this.cancelAnimations()
-    this.setState({ spinner: true }, () => {
-      this.props.dispatch(
-        getApplicationState(() => {
-          this.animateCloseTimeout()
-        })
-      )
-    })
+    this.setState({ spinner: true })
   }
 
-  animateCloseTimeout() {
+  // TODO - trigger this when load is done
```

Comment:
We are pushing to the right route, is this no longer the place that needs to happen?

Reply:
The remaining TODO here is the spinner used to animate once the loader was done, _before_ routing to the next page. I'm not sure how important keeping it is to the project, but I wanted to call it out since it is the only deprecation of current functionality in this PR.

Reply:
oh right, there used to be a minimum 1s delay or something before moving on? That is probably a nice thing to have design wise